packages/scripts/environments/staging.yml

```diff
@@ -86,6 +86,27 @@ cronJobs:
         memory: 5Gi
     backoffLimit: 2
 
+  - name: process-citations-for-metrics-dev
+    schedule: "30 0 * * *" # every day at 00:30 (12:30 AM) UTC
```

Comment:
Any reason for using a different time than in prod? Probably doesn't really matter but might be nice to be consistent.